src/Macroscope/Worker.hs

```diff
@@ -105,12 +100,11 @@ processStream logFunc postFunc = go (0 :: Word) [] []
         -- The end of the stream
         res <- processBatch acc
         pure $ reverse (res : results)
-      Right (Left err, rest) -> do
-        -- An error occured in the stream, abort now
-        let res = Just (StreamError (err, rest))
```

Comment:
We have test that fail, I guess that's because now we commit the date. Should we update the test and make sure we get a Error doc instead ?

Reply:
Yes, though we need to design the error query first.